services/user/CommonSys/common-lib/packages/wasm-loader/tsconfig.json

```diff
@@ -3,13 +3,13 @@
     "target": "ES2020",
```

Comment:
We need to make sure we're formatting these correctly too...and that `.prettierrc` files are all where they need to be.